docs/commands/super.md

```diff
@@ -947,344 +969,595 @@ Sure enough, this works:
 ```
 duckdb gha.db -c "CREATE TABLE gha AS FROM read_json('gharchive_gz/*.json.gz', union_by_name=true)"
 ```
-We now have the `duckdb` database file for our GitHub Archive data called `gha.db`
+We now have the DuckDB database file for our GitHub Archive data called `gha.db`
 containing a single table called `gha` embedded in that database.
 What about the super-structured
 format for the `super` command?  There is no need to futz with sample sizes,
-schema inference, or union by name, just run this to create a Super Binary file:
+schema inference, or union by name. Just run this to create a Super Binary file:
 ```
 super gharchive_gz/*.json.gz > gha.bsup
 ```
 
 ## Appendix 2: Running the Tests
 
-This appendix provides the raw tests and output that we run on a MacBook Pro to generate
-the table of results above.
+This appendix provides the raw tests and output from the [most recent archived run](https://super-cmd-perf.s3.us-east-2.amazonaws.com/2024-11-26_03-17-25.tgz)
+of the tests via [automated scripts](https://github.com/brimdata/super/blob/main/scripts/super-cmd-perf)
```

Comment:
Since the scripts currently only exist on this branch, this hyperlink will fail until this PR is merged.